AWS.md

```diff
@@ -1,7 +1,18 @@
 General
 -------
 
- * Use CloudFormation or ElasticBeanStalk to define infrastructure in AWS.
+ * Use [CDK](https://github.com/guardian/cdk) CloudFormation or ElasticBeanStalk to define infrastructure in AWS
+   * Prefer to [import resources](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import.html) into CFN over manually configuring them
+ * Tag resources with:
+   * `Stack` - the broad umbrella the service sits under e.g. `deployTools`
```

Comment:
Minor: we tend to use `deploy` as the stack name

Reply:
Ah yes. I might choose a different example as a `deploy` stack isn't particularly obvious...

Reply:
Giving a good example here is tricky... 

I think 'Stack' provides the most benefits when we use it to mean _a logical grouping of microservices that are responsible for some particular functionality_ e.g. `mobile-notifications`. For example, this allows you to easily filter all logs responsible for sending a notification, regardless of which app/lambda is doing the work. But in practice for most teams `Stack` is synonymous with _the account that you want to deploy things into_.

Reply:
Yeah. I was hoping "umbrella" was generic/non-specific enough to cater for both of these use-cases. But maybe that's not helpful?

We've discussed before, but `Stack` doesn't really make sense anymore for the reasons you describe and also because it's overloaded.

Reply:
I think the updated example makes this as clear as possible 👍  (I'd like to remove the concept of `Stack` entirely but that's slightly outside the scope of this PR 🙃 )